lib/onelogin/ruby-saml/attribute_service.rb

```diff
@@ -14,18 +19,29 @@ def configure(&block)
         instance_eval &block
       end
 
+      # @return [Boolean] True if the AttributeService object has been initialized and set with the required values
+      #                   (has attributes and a name)
       def configured?
         @attributes.length > 0 && !@name.nil?
       end
 
+      # Set a name 
+      # @param name [String] The service name
+      #
       def service_name(name)
         @name = name
       end
 
+      # Set an index 
+      # @param name [Integer] An index
+      #
       def service_index(index)
```

Comment:
I was only adding comments on this PR, the code on master is without the =.

Notice that the attribute is index, and we named the setter service_index.